jib-core/src/test/java/com/google/cloud/tools/jib/api/JibContainerBuilderTest.java

```diff
@@ -362,4 +363,23 @@ public void setApplyContainerBuildPlan()
     Assert.assertEquals(
         ImmutableSet.of(new Platform("testArchitecture", "testOS")), convertedPlan.getPlatforms());
   }
+
+  @Test
+  public void testContainerize_multiPlatformsList()
+      throws InvalidImageReferenceException, CacheDirectoryCreationException, InterruptedException,
+          RegistryException, IOException, ExecutionException {
+    try {
+      ImageConfiguration imageConfiguration =
+          ImageConfiguration.builder(ImageReference.parse("base/image")).build();
+      new JibContainerBuilder(imageConfiguration, spyBuildContextBuilder)
+          .setPlatforms(
+              ImmutableSet.of(
+                  new Platform("testArchitecture", "testOS"),
+                  new Platform("testArchitecture1", "testOS2")))
+          .containerize(Containerizer.to(RegistryImage.named("target/image")));
```

Comment:
Generally you should limit the scope of `try { } catch` for this type of exception-checking tests, because theoretically other code (e.g., `ImageConfiguration.build()`) may also throw `UnsupportedOperationException`. We are checking the exception message which is very specific, but in the worst case, it's possible that a test becomes useless to catch a bug if the test doesn't check the message or the message is very generic.

```java
      ImageConfiguration imageConfiguration =
          ImageConfiguration.builder(ImageReference.parse("base/image")).build();
      JibContainerBuilder containerBuilder = new JibContainerBuilder(imageConfiguration, spyBuildContextBuilder)
          .setPlatforms(
              ImmutableSet.of(
                  new Platform("testArchitecture", "testOS"),
                  new Platform("testArchitecture1", "testOS2")))
    try {
          containerBuilder.containerize(Containerizer.to(RegistryImage.named("target/image")));
```

and I guess `new Platform("arch1", "os1"), new Platform("arch2", "os2")` could save a few lines (which isn't important at all but I think it's an easy way to save some of the expensive screen-estate).

Reply:
Updated the code.

Reply:
Good Catch